lib/scope.js

```diff
@@ -1065,6 +1065,27 @@ module.exports = {
         await scope.uploadFiles( scope, { handle, set_to });
         await scope.afterStep(scope, { waitForShowIf: true });
 
+      } else if (type == `date`) {
+        let set_to_date = set_to;
+        // If the date is specified in the format "today" or "today+2" or "today-1"
+        // it will need some pre-processing. 
+        if (set_to.indexOf(`today`) !== -1) {
```

Comment:
Just realized the developer may use spaces between some characters.
```suggestion
        if (set_to.indexOf(`today`) !== -1) {
          set_to = set_to.replace(/\s/g, '');
```